app/components/modals/AddMixerAccountsModal.jsx

```diff
@@ -1,14 +1,26 @@
-import { FormattedMessage as T } from "react-intl";
+import { FormattedMessage as T, injectIntl, defineMessages } from "react-intl";
 import { TextInput, PassphraseModalField } from "inputs";
 import { PassphraseModal } from "modals";
 import { useMountEffect } from "hooks";
 
+const messages = defineMessages({
+  mixedAccountName: {
+    id: "addMixerAccountModal.mixedAccountName",
```

Comment:
The convention I've seen in other files is to suffix the IDs here with `.placeholder`.

For example...

https://github.com/decred/decrediton/blob/683e3658b8e26ed6f3f6979b162acf4c25c99a0c/app/components/views/PrivacyPage/SecurityPage/VerifyMessage/VerifyMessageForm.jsx#L8-L21

Reply:
nice suggestion! I haven't paid attention to that pattern, sorry!